core/src/main/scala/org/apache/spark/scheduler/AsyncEventQueue.scala

```diff
@@ -169,7 +169,8 @@ private class AsyncEventQueue(
           val prevLastReportTimestamp = lastReportTimestamp
           lastReportTimestamp = System.currentTimeMillis()
           val previous = new java.util.Date(prevLastReportTimestamp)
-          logWarning(s"Dropped $droppedCount events from $name since $previous.")
+          logWarning(s"Dropped $droppedCount events from $name since " +
+            s"${if (prevLastReportTimestamp == 0) "start of the application" else s"$previous"}.")
```

Comment:
nit. "start of the application" -> "the application starts", @shivusondur ?

Reply:
@dongjoon-hyun Done.